components/Header/Header.js

```diff
@@ -84,7 +84,7 @@ class Header extends Component {
           <AccountDropdown />
           <MiniCart />
         </Toolbar>
-        <NavigationMobile />
+        <NavigationMobile shop={shop}/>
```

Comment:
I saw an issue similar to this Re: sitemaps in a different project. Neither issue would get picked up by TypeScript static analysis if the project were converted. As @janus-reith mentioned in #709 a conversion may not be suitable for front-end until other repos in the org are in line with a conversion initiative.